keyboards/kakunpc/rabbit_capture_plan/keyboard.json

```diff
@@ -51,6 +51,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
```

Comment:
Missing rules.mk changes here?

Reply:
Seems zvecr and I have been duplicating work again.